internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -273,6 +287,53 @@ func (opts *Opts) askDBUserAccessListOptions() error {
 	return nil
 }
 
+func (opts *Opts) askMongoShellQuestion() (bool, error) {
+	if opts.SkipMongoShellQuestions {
+		return false, nil
+	}
+
+	if opts.ConfigMongoShellPath() == "" {
+		answer := false
+		prompt := newMongoShellQuestionNotFound()
+		if err := survey.AskOne(prompt, &answer); err != nil {
+			return false, err
+		}
+
+		if answer {
+			if err := askMongoShellAndSetConfig(); err != nil {
+				return false, err
+			}
+		} else {
+			openURL := false
+			prompt := newMongoShellQuestionBrowser()
+			if err := survey.AskOne(prompt, &openURL); err != nil {
+				return false, err
+			}
+
+			if openURL {
```

Comment:
[nit] this is the 4th `if` level here, let's try to clean this up a little to either other functions on some better pattern, more than one level deep starts to smell 4 levels deep definitely can be improved